src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldExtendedLookup.class.php

```diff
@@ -152,6 +150,58 @@ protected function _GetOpenWindowJS(&$oPopupTableConf)
         return $js;
     }
 
+    private function getTargetListRestriction(): array
+    {
+        $restrictionExpression = $this->oDefinition->GetFieldtypeConfigKey('restriction');
+        if (null === $restrictionExpression) {
+            return [];
+        }
+        if (false === \strpos($restrictionExpression, '=')) {
+            return [];
+        }
+        $equalPosition = \mb_strpos($restrictionExpression, '=');
+        $restrictionField = \trim(\mb_substr($restrictionExpression, 0, $equalPosition));
+        $restrictionValue = \trim(\mb_substr($restrictionExpression, $equalPosition + 1));
+
+        if (\mb_strpos($restrictionValue, '[{') === 0 && '}]' === \mb_substr($restrictionValue, -2)) {
+            $restrictionValueReplaced = $this->getFieldValueFromFieldToken($restrictionValue);
+            if (null === $restrictionValueReplaced) {
+                $restrictionValueReplaced = \sprintf('field token %s not found in record.', $restrictionValue);
```

Comment:
This is then simply shown in the URL?

Reply:
Changed to now show a toaster error message as well